bin/cli.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+set -o noclobber
+set -o errexit
+set -o pipefail
+set -o nounset
+
+FONTS_DIRECTORY="fonts"
+RELEASE_DATE=$(date +'%B %d, %Y')
+STYLE="owncloud"
+STYLES_DIRECTORY="resources/themes"
+
+ERR_UNSUPPORTED_LANGUAGE=20
+ERR_UNSUPPORTED_MANUAL=21
+ERR_UNSUPPORTED_ACTION=22
+
+if [[ -z $VERSION ]]; then
+    if [[ -n $DRONE_TAG ]]; then
+        VERSION=${DRONE_TAG/v//}
+    else
+        [[ -n $DRONE_BRANCH ]] && VERSION=${DRONE_BRANCH/v//} || VERSION=master
+    fi
+fi
+
+function usage()
+{
+    echo "Usage: cmd [-c] [-b <admin|developer|user>] [-v <go|xml|json|php|kotlin|yaml>] -r <revision> -d <release date>"
+}
+
+function clean_build_dir()
+{
+    echo "Cleaning build directory..."
+    rm -rvf "build/"
+    echo "...build directory cleaned."
+}
+
+function validate_code_files()
+{
+    language="$1"
+
+    case $language in
+        go)
+            command -v golint &>/dev/null && \
+                echo "Validating go source files" && \
+                find . -type f -name "*.go" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && golint {} && echo' \;
+            ;;
+
+        json)
+            command -v jsonlint &>/dev/null && \
+                find . -type f -name "*.json" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && jsonlint -qp {} && echo' \;
+            ;;
+
+        kotlin)
+            command -v ktlint &>/dev/null && \
+                ktlint --reporter=plain "./modules/*_manual/**/*.kt" || true;
+            ;;
+
+        php)
+            command -v php &>/dev/null && \
+                find ./modules/*_manual/examples -type f -name "*.php" -exec php -l {} \;
+            ;;
+
+        xml)
+            command -v xmllint &>/dev/null && \
+                find ./modules/*_manual/examples -type f -name "*.xml" -exec xmllint --noout {} \;
+            ;;
+
+        yaml)
+            command -v yamllint &>/dev/null && \
+                find . -type f -name "*.yml" \
+                    ! -path "./node_modules/*" \
+                    ! -path "**/vendor/*" \
+                    ! -path "./.git/*" \
+                    -exec sh -c 'echo Linting {} && yamllint -f parsable {} && echo' \;
+            ;;
+
+        *)
+            echo "That language is not, currently, supported" 
+            exit $ERR_UNSUPPORTED_LANGUAGE
+            ;;
+    esac
+}
+
+function convert_antora_nav_to_asciidoc_list()
+{
+    filename="$1"
+
+    while read line 
+    do
```

Comment:
You put `if` on a single line but not this loop?

Reply:
If it's a better approach to do that here, fine by me.